package.json

```
@@ -22,8 +22,8 @@
         "yargs": "^17.0.0"
     },
     "devDependencies": {
-        "@axe-core/cli": "^4.8.0",
-        "@axe-core/puppeteer": "^4.8.0",
+        "@axe-core/cli": "^4.9.1",
+        "@axe-core/puppeteer": "^4.9.1",
```

These are usually specified using the minor version number, with patch version at 0
```suggestion
        "@axe-core/cli": "^4.9.0",
        "@axe-core/puppeteer": "^4.9.0",
```